README.md

```diff
@@ -1,2 +1,36 @@
-# basic-auth
-401-lab-06
+# LAB - Class 06
+
+## Project: Basic Auth
+
+### Author: Simon Panek
+
+#### Contributors: Ricardo, Nathan, Mariko, Tina, & Garrett
+
+### Links and Resources
+
+- [GitHub Actions ci/cd](https://github.com/simon-panek/basic-auth/actions)
+- [Heroku Deployment](https://simonpanek-basic-auth.herokuapp.com/)
+
+### Setup
+
+#### `.env` requirements (where applicable)
+
+- `PORT` - 3000
+- `MONGODB_URI` - mongodb://localhost:27017/auth
```

Comment:
Make sure that this is the one you actually use!

Reply:
Thanks, good catch!